src/Microsoft.AspNetCore.Mvc.Formatters.Xml/Internal/MvcXmlDataContractSerializerMvcOptionsSetup.cs

```diff
@@ -42,6 +43,16 @@ public void Configure(MvcOptions options)
             options.OutputFormatters.Add(new XmlDataContractSerializerOutputFormatter(_loggerFactory));
             options.InputFormatters.Add(new XmlDataContractSerializerInputFormatter(options));
 
+            // Do not override any user mapping
+            var key = "xml";
+            var mapping = options.FormatterMappings.GetMediaTypeMappingForFormat(key);
+            if (string.IsNullOrEmpty(mapping))
+            {
+                options.FormatterMappings.SetMediaTypeMappingForFormat(
+                    key,
+                    MediaTypeHeaderValue.Parse("application/xml"));
```

Comment:
Regarding these literals, do we always just hardcode these? Isn't there a set of constants for those?

Reply:
Good point. I found that we already have something similar that I could use.